src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/BlogSummaryMailGenerator.java

```diff
@@ -42,10 +41,10 @@ public BlogSummaryMailGenerator() {
 
     @Autowired
     public BlogSummaryMailGenerator(BlogRepository blogRepository,
-                                    BlogPostRepository blogPostRepository,
-                                    SettingRepository settingRepository,
-                                    NowProvider nowProvider,
-                                    SyndFeedProducer syndFeedFactory) {
+        BlogPostRepository blogPostRepository,
```

Comment:
I will rather try not introduce a change that brings a reformatting only. In your case it's not a biggy but for large files it may obscure diff readability.